pages/premium.php

```diff
@@ -578,8 +578,13 @@ function premium() {
                 // Confirm change in seal... time will not be reimbursed
                 if(!isset($_POST['change_forbidden_seal'])) {
                     $confirmation_type = 'change_forbidden_seal';
+                    // Convert remaining premium time to days and calculate AK value
+                    $akCredit = $player->forbidden_seal->calcRemainingCredit();
+                    // Adjust purchase cost with minimum 0
+                    if (($akCost -= $akCredit) < 0) {$akCost = 0;}
```

Comment:
I know it's valid syntax to do assignments inside if statements, but I'd rather avoid those wherever possible and make the intention very clear
```suggestion
                    $akCost -= $akCredit;
                    if ($akCost < 0) {
                       $akCost = 0;
                     }
```